app/assets/stylesheets/base.scss

```diff
@@ -9,6 +9,12 @@ a {
     fill: $sul-link-color;
   }
 
+  &.btn {
```

Comment:
If you haven't, I'd try rebasing and see if this is still necessary. There was a fix recently that should auto-calculate the right icon color for a given button type.

Reply:
Removed the extra CSS.